### DISCUSSION TITLE
Welcome to ddt4all Discussions!

### DISCUSSION THREAD

**Discussion by cedricp** [cedricp/ddt4all#832] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Furtif** [comment]
💯

**Reply by shrlnm** [reply]
Hi!   It's a good idea to go to the discussion section.

**Reply by Furtif** [reply]
yup ... you is the pyren manager in gitlab if i m right ??

**Reply by Furtif** [reply]
do not be disturbed I like the exchange of ideas

**Reply by baneks83** [reply]
hello all, how to change the gearbox type on the ecu sid 305 when the program does not give me the configuration option. do you have any ideas or an xml file for the sid 305 megan 3

**Comment by NathanScott89** [comment]
Does DDT4ALL with ELM the DoIP Protokoll?

**Reply by Furtif** [reply]
DoIP is not supported by dd4all so far

**Reply by Furtif** [reply]
within 5 years DDT4ALL will only support collection cars ... boy (or girl) it all depends on the database used, not just 'this'

**Comment by LemurMort** [comment]
Hello all!
I don't want to mess something UP with decimale values... 
This is a c/p of issue #823

I have a question regarding duration of automatic re-lock (if after unlocking the car no door is opened the car locks itself again).
Default value is duration in minutes (0, 1, 2...).

I'm curious if decimal values can be put here.
Ie. for 30s should the value be 0.5 or 0,5 (with dot or comma)?

Thank you!

**Comment by Furtif** [comment]


**Reply by MikeRS265** [reply]
How do i get the ddt2000 database ???
Ive been looking for 1 hour now and havent found anywhere to download it...

**Comment by Furtif** [comment]


**Reply by Renotech1** [reply]
Does this still works?
Would be great if we can just download newest database like rafale/renault duo bento ect ect
Probably there are new server adresses for this...

**Comment by Chumneywarner** [comment]
Hi,

I am new to this, so please excuse any ignorance, but I am trying to learn.

I have a problem with the latest versions or DDT4all.
It installs and loads OK and I can select my vehicle, the unit I want to investigate in the top left window, then drill down to the next item in the middle left window, but when I try to select the item I want to adjust in the lower left window, the program crashes and I end up back at my Windows desktop.

I have installed DDT4all 23.06.13-python-3.11.4-64bits.

If I uninstall DDT4all 23.06.13-python-3.11.4-64bits and reinstall using DDT4all 23.06.13-python-3.8.6-32bits, the same thing happens, 

I have tried using the fix suggested by drder66 and have replaced the code as advised (I believe I have done this correctly), but I still have the same issue in both versions.
I have also tried installing VCredistx64, but no improvement.
I am also updated the ECU database and am now using the latest DDT2000 ECU database complied by Furtif.

My car is a 2017 Megane IV (I believe XFB), 1.5 diesel automatic.
I am using Windows 11 Pro on a 64 bit machine on a x64 machine, so obviously I would prefer to use the latest x64 version of DDT4all.

Any ideas on a fix for this issue please?

Thanks
Chumneywarner

**Comment by gigen1979** [comment]
hallo, I'm Franco from Italy and I have 2016 duster dci and need help with DDT4ALL

**Comment by Alecsande1985** [comment]
Добрый день, Кто-нибудь встречался с установкой радар дистроник и его подключением, если у кого что-то есть поделитесь пожалуйста?

**Comment by baneks83** [comment]
hello all, how to change the gearbox type on the ecu sid 305 when the program does not give me the configuration option. do you have any ideas or an xml file for the sid 305 megan 3

**Comment by ismata18** [comment]
Hello. I cannot find the ecu.zip file for 2025 Dacia Sandero Stepway. Thank you
![Uploading IMG_20260126_170919.jpg…]()
